content/zh-cn/blog/_posts/2022-08-04-kubernetes-1.24-deprecations-and-removals.md

```diff
@@ -0,0 +1,193 @@
+-->
+随着 Kubernetes 成长和日趋成熟，为了此项目的健康发展，某些功能特性可能会被弃用、移除，
+或替换为优化过的功能特性。Kubernetes V1.25 包括几个主要变更和一个主要移除。
+
+<!--
+## The Kubernetes API Removal and Deprecation process
+
+The Kubernetes project has a well-documented [deprecation policy](/docs/reference/using-api/deprecation-policy/) for features. This policy states that stable APIs may only be deprecated when a newer, stable version of that same API is available and that APIs have a minimum lifetime for each stability level. A deprecated API is one that has been marked for removal in a future Kubernetes release; it will continue to function until removal (at least one year from the deprecation), but usage will result in a warning being displayed. Removed APIs are no longer available in the current version, at which point you must migrate to using the replacement.
+-->
+## Kubernetes API 移除和弃用流程 {#the-kubernetes-api-removal-and-deprecation-process}
+
+Kubernetes 项目对功能特性有一个文档完备的[弃用策略](/zh-cn/docs/reference/using-api/deprecation-policy/)。
+该策略规定，只有当较新的、稳定的相同 API 可用时，原有的稳定 API 才可能被弃用，每个稳定级别的 API 都有一个最短的生命周期。
+弃用的 API 指的是已标记为将在后续发行某个 Kubernetes 版本时移除的 API；
+移除之前该 API 将继续发挥作用（从弃用起至少一年时间），但使用时会显示一条警告。
+移除的 API 将在当前版本中不再可用，此时你必须迁移以使用替换的 API。
+
+<!--
+* Generally available (GA) or stable API versions may be marked as deprecated but must not be removed within a major version of Kubernetes. 
+* Beta or pre-release API versions must be supported for 3 releases after deprecation. 
+* Alpha or experimental API versions may be removed in any release without prior deprecation notice.
+-->
+* 正式发布（GA）或稳定的 API 版本可能被标记为已弃用，但只有在 Kubernetes 大版本更新时才会移除。
+* 测试版（Beta）或预发布 API 版本在弃用后必须支持 3 个版本。
+* Alpha 或实验性 API 版本可能会在任何版本中被移除，恕不另行通知。
+
+<!--
+Whether an API is removed as a result of a feature graduating from beta to stable or because that API simply did not succeed, all removals comply with this deprecation policy. Whenever an API is removed, migration options are communicated in the documentation.
+-->
+无论一个 API 是因为某功能特性从 Beta 进入稳定阶段而被移除，还是因为该 API 根本没有成功，
+所有移除均遵从上述弃用策略。无论何时移除一个 API，文档中都会列出迁移选项。
+
+<!--
+## A Note About PodSecurityPolicy
+
+In Kubernetes v1.25, we will be removing PodSecurityPolicy [after its deprecation in v1.21](/blog/2021/04/06/podsecuritypolicy-deprecation-past-present-and-future/). PodSecurityPolicy has served us honorably, but its complex and often confusing usage necessitated changes, which unfortunately would have been breaking changes. To address this, it is being removed in favor of a replacement, Pod Security Admission, which is graduating to stable in this release as well. If you are currently relying on PodSecurityPolicy, follow the instructions for [migration to Pod Security Admission](/docs/tasks/configure-pod-container/migrate-from-psp/).
+-->
+## 有关 PodSecurityPolicy 的说明 {#a-note-about-podsecuritypolicy}
+
+[继 PodSecurityPolicy 在 V1.21 弃用后](/blog/2021/04/06/podsecuritypolicy-deprecation-past-present-and-future/)，
+Kubernetes V1.25 将移除 PodSecurityPolicy。PodSecurityPolicy 曾给我们以荣耀，
```

Comment:
Not sure if it's intended, but I would more prefer `v1.25` format than `V1.25` format.

```suggestion
[继 PodSecurityPolicy 在 v1.21 弃用后](/blog/2021/04/06/podsecuritypolicy-deprecation-past-present-and-future/)，
Kubernetes v1.25 将移除 PodSecurityPolicy。PodSecurityPolicy 曾给我们以荣耀，
```

Reply:
ok, changed back. It's a good idea and easy way to follow the current style. 😄